gmail/1.1/modules/ROOT/pages/gmail-connector-reference.adoc

```diff
@@ -5117,7 +5117,7 @@ Set up or update a push notification watch on the given user mailbox. This opera
 `<gmail:send-email>`
 
 
-Sends the specified message to the recipients in the To, Cc, and Bcc headers. This operation makes an HTTP POST request to the /gmail/v1/users/{userId}/messages/send endpoint.
+Sends the specified message to the recipients in the To, Cc, and Bcc headers. This operation makes an HTTP POST request to the /gmail/v1/users/{userId}/messages/send endpoint and returns the email ID of the sent message.
```

Comment:
Seems like this is the Composer operation that somehow appears in the generated reference (despite being marked with Ignore annotation). I think we should remove this Send email operation because users cannot find it in Studio and it can be confusing. What do you think?